### PR TITLE
Update icons to version 2.0.13

### PR DESCRIPTION
**Pull request by jprider63**
I updated the generator for the latest heroicon version and repository structure.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "dioxus-heroicons"
-version = "0.1.4"
+version = "2.0.13"
```
Comment (autarch) [inline Cargo.toml:3]: I don't think it makes sense to change the crate version to match the heroicons version, at least not right now. The crate has only had a few releases, so I don't want to give it a stable version number yet.

File: gen/src/main.rs [hunk 2]
```diff
@@ -25,15 +25,25 @@ struct Icon {
...
     let args = Args::parse();
 
     let mut src_dir = args.heroicons.clone();
     src_dir.push("src");
 
-    for style in &["outline", "solid"] {
+    for style in &["outline", "solid", "mini"] {
         let mut src_dir = src_dir.clone();
-        src_dir.push(style);
+
```
Comment (autarch) [inline gen/src/main.rs:45]: Please don't add whitespace changes like this.

File: gen/src/main.rs [hunk 3]
```diff
@@ -121,18 +131,18 @@ impl crate::IconShape for Shape {
 "#;
 
 const PATH_TEMPLATE: &str = r#"
-Shape::{NAME} => rsx! {
-    path {
-        {ATTRS}
-    },
-},"#;
+            Shape::{NAME} => rsx! {
```
Comment (autarch) [inline gen/src/main.rs:134]: Ditto here re: whitespace changes.

File: gen/src/main.rs [hunk 4]
```diff
@@ -121,18 +131,18 @@ impl crate::IconShape for Shape {
...
+                },
+            },"#;
 
 fn write_icons_file(icons: &[Icon], to: &PathBuf) {
     let names = icons
         .iter()
         .map(|i| i.name.as_str())
         .collect::<Vec<_>>()
-        .join(",\n");
+        .join(",\n    ");
```
Comment (autarch) [inline gen/src/main.rs:145]: All of the generated code will be cleaned up with `rustfmt` anyway so there's no need to worry about whitespace in that code.

**Review by autarch — CHANGES_REQUESTED** [review]
Thanks for this! This looks generally good except for a few small issues.

**Comment by jprider63** [comment]
Should be good now. I didn't realize you were running `rustfmt` to fix the whitespace. Let me know if you'd like me to make any other changes.

**Comment by autarch** [comment]
Hi, thanks for your PR! I'm pretty finicky about my projects (see [this blog post](https://blog.urth.org/2022/07/02/what-s-the-right-way-to-merge-a-pull-request/) for details), so I rarely merge a PR as-is. I can move forward on your PR in one of two ways:

1. I check it out locally, fiddle with it as needed, merge it locally, and simply close this PR. This will preserve at least one commit with your name on it, but the PR will show up as closed in your GitHub stats.
2. If you enable me to push directly to your fork, I can do my fiddling, then force push to your fork and merge the resulting PR. Again, this will preserve at least one commit with your name on it, but you _also_ get credit for the PR merge in your GitHub stats. The only downside is that I will be force pushing directly to your fork.

Please let me know which approach you'd prefer. If I don't hear from you before I get around to working on this PR I'll go with option #1.

Thanks again for your contribution!

**Comment by jprider63** [comment]
Either works for me. You should have permission for 2 (I checked allow edits by maintainers). Let me know if you have any trouble.

**Comment by autarch** [comment]
It's not letting me push to your branch for some reason. You're sure you checked the box?

**Comment by jprider63** [comment]
Yup, it's checked. No worries if it's not working.

**Comment by autarch** [comment]
Very weird. I'll just go with method 1.

**Comment by autarch** [comment]
Merged from the CLI.

**Pull request closed without merging**